src/drafts/InlineAutocomplete/InlineAutocomplete.tsx

```diff
@@ -79,7 +79,6 @@ const InlineAutocomplete = ({
   sx,
   children,
   tabInsertsSuggestions = false,
-  // Forward accessibility props so it works with FormControl
   ...forwardProps
```

Comment:
Forwarding isn't necessary to make it work with `FormControl` anymore since the underlying `TextInput` or `Textarea` will handle it